File: app/src/main/java/heyalex/com/miet_schedule/model/schedule/Times.java
package heyalex.com.miet_schedule.model.schedule;

/**
 * Contains information about schedule time exercise
 * Part of {@link SemesterData}
 */
/*package*/ class Times {

    /**
     * Number of exercise in all schedule time
     */
    private String Time;

    /**
     * Lesson time start
     */
    private String TimeFrom;

    /**
     * Lesson time end
     */
    private String TimeTo;

    private String Code;

    public String getTime() {
        return Time;
    }

    public void setTime(String Time) {
        this.Time = Time;
    }

    public String getTimeFrom() {
        return TimeFrom;
    }

    public void setTimeFrom(String TimeFrom) {
        this.TimeFrom = TimeFrom;
    }

    public String getTimeTo() {
        return TimeTo;
    }

    public void setTimeTo(String TimeTo) {
        this.TimeTo = TimeTo;
    }

    public String getCode() {
        return Code;
    }

    public void setCode(String Code) {
        this.Code = Code;
    }
}

File: miet-api/src/main/java/com/alex/miet/miet_api/model/Times.kt
package com.alex.miet.miet_api.model

import com.google.gson.annotations.SerializedName

data class Times(

    @SerializedName("Code") val code: Int,
    /**
     * Number of exercise in all schedule time
     */
    @SerializedName("Time") val time: String,
    /**
     * Lesson time start
     */
    @SerializedName("TimeFrom") val timeFrom: String,

    /**
     * Lesson time end
     */
    @SerializedName("TimeTo") val timeTo: String
)